src/main/java/org/zalando/nakadi/repository/db/StorageDbRepository.java

```diff
@@ -0,0 +1,74 @@
+package org.zalando.nakadi.repository.db;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+import org.zalando.nakadi.annotations.DB;
+import org.zalando.nakadi.domain.Storage;
+
+@DB
+@Repository
+public class StorageDbRepository extends AbstractDbRepository {
+    @Autowired
+    public StorageDbRepository(final JdbcTemplate template, final ObjectMapper mapper) {
+        super(template, mapper);
+    }
+
+    public List<Storage> listStorages() {
```

Comment:
I thought about this, but decided not to do that. In fact, if we will have more than 10 active kafka clusters, I will be really surprised. If the cluster is not used, then we have to delete it to avoid mistakes while timelines creation